pkg/gds/certs.go

```diff
@@ -53,14 +53,14 @@ func (s *Service) CertManager(stop <-chan struct{}) {
 		case <-ticker.C:
 		}
 
-		if err := s.HandleCertifcateRequests(certDir); err != nil {
+		if err := s.HandleCertificateRequests(certDir); err != nil {
 			log.WithLevel(zerolog.PanicLevel).Err(err).Msg("could not handle certificate requests, certificate manager shutting down")
 			return
 		}
 	}
 }
 
-func (s *Service) HandleCertifcateRequests(certDir string) (err error) {
+func (s *Service) HandleCertificateRequests(certDir string) (err error) {
```

Comment:
Good typo catch! It took me a minute to see it even with the diff in front of me!